### PR TITLE
Chore/added sample env

### PR DESCRIPTION
**Pull request by JohnAllenTech**
# ✨ Codu Pull Request 💻

Codu Logo

## Pull Request details

- Small PR to add an example .env file
- Took me a minute to read through the README and figure out the values needed for .env
- I think added a sample and linking to it in the README is a good idea as it will give new contributors a sanity check

## Any Breaking changes

- None

## Associated Screenshots

- None

## Summary by CodeRabbit

- **Documentation**
    - Enhanced installation instructions with dependency setup and environment variable configuration details.
    - Clarified database setup and seeding steps.
    - Provided a sample `.env` file for easy reference.
    - Updated and improved contribution guidelines section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,31 +22,39 @@ We have a fantastic community growing on Discord. Click [here](https://discord.g
...
-  DATABASE_URL=postgresql://postgres:secret@127.0.0.1:5432/postgres
-  # Setup your GitHub ID & Secret on GitHub: https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps
-  # For development, make sure you set up this with a Homepage URL of http://localhost:3000/ and an Authorization callback URL of http://localhost:3000/api/auth
-  GITHUB_ID=YOUR_GITHUB_APP_ID
-  GITHUB_SECRET=YOUR_GITHUB_APP_SECRET
-  NEXTAUTH_URL=http://localhost:3000/api/auth
-  ```
-
-  For a more detailed how to guide on setting them up [go to the Environment Variables section](#environment-variables).
+
```
Comment (JohnAllenTech) [inline README.md:33]: No idea why its adding this line?  I didnt change this. Will look into my prettier settings as maybe I have a global config but it should be pulling from the repo config.

File: README.md [hunk 2]
```diff
@@ -105,11 +113,11 @@ NEXTAUTH_URL=http://localhost:3000/api/auth
 ```
 
 For more information, you can read the documentation [here](https://next-auth.js.org/configuration/options).
+**Example .env file can be found [here](./sample.env). You can rename this to .env to get started**
```
Comment (JohnAllenTech) [inline README.md:116]: This is all that I added to the README. Perhaps it would be better at the top. Not sure

File: README.md (hunk 1 above)
Comment (NiallJoeMaher) [reply README.md:33]: Not a big deal I think anyway!

**Review by NiallJoeMaher — APPROVED** [review]
🌮 LGTM! Always good to get the docs clearer!

**Pull request merged**